sound/sample_sub_audioclass.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+from collections import deque
+import rospy
+import math
+import time
+from std_msgs.msg import String
+from netcat import netcat
+
+'''
+   This is a sample subscriber written to check the class of AudioInput.
```

Comment:
Is this a sample, or an example?  The description makes it sound like an example ... that is -- "This is an example of a subscriber that checks the AudioInput class"  

unless you actually mean "sample" ... but I don't think you do.  Please fix the comment, and change the name of the file to reflect this.